tools/gen-manifests

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Generate manifests using the gen-manifests tool in github.com/osbuild/images
+
+set -euo pipefail
+
+workdir="$(pwd)"
+
+# get the version tag of osbuild/images from the go.mod
+images_version="$(go list -m -json -versions -mod=readonly github.com/osbuild/images | jq -r .Version)"
+
+tmpdir="$(mktemp -d)"
+
+cleanup() {
+    rm -rf "${tmpdir}"
+}
+
+trap cleanup EXIT
+
+# shallow clone osbuild/images into a temporary directory to generate the manifests
+echo "Cloning osbuild/images ${images_version}"
+git clone https://github.com/osbuild/images --branch "${images_version}" --depth=1 "${tmpdir}"
+pushd "${tmpdir}"
+
+manifest_dir="${workdir}/test/data/manifests/"
+echo "Generating manifests and storing in ${manifest_dir}"
+go run ./cmd/gen-manifests --output "${manifest_dir}"
```

Comment:
(super nitpick) I think this could be `go run github.com/osbuild/images/cmd/gen-manifests@latest`  (and the clone above would be no longer needed nor the cleanup of tmpdir

Reply:
We could have separate configs and repos for osbuild-composer, ones that would match the things we need or want to test here so they're separate from osbuild/images, but I think that would cause a lot of duplication and drift again, and the whole change in this PR is a (hopefully) temporary solution.

Reply:
Also, I don't feel comfortable running the `latest` version all the time, but that would be possible to solve by using the version tag instead.

Reply:
Aha, sure! Sorry, missed that detail.